buildscripts/gateway-tests.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+set -e
+
+export SERVER_ENDPOINT=127.0.0.1:24240
+export ENABLE_HTTPS=0
+export ACCESS_KEY=minio
+export SECRET_KEY=minio123
+export MINIO_ACCESS_KEY=minio
+export MINIO_SECRET_KEY=minio123
+export AWS_ACCESS_KEY_ID=minio
+export AWS_SECRET_ACCESS_KEY=minio123
+
+./minio server data --address 127.0.0.1:24242 &
+sleep 3
+./minio gateway s3 http://127.0.0.1:24242 --address 127.0.0.1:24240 &
+sleep 3
+
+mkdir -p /mint
+git clone https://github.com/minio/mint /mint
+cd /mint
```

Comment:
This can be just 

```
/mint/release.sh
```

Reply:
I want to break it up further actually.

`/mint/release.sh` either builds all packages or none at all. I want to achieve functionality where only updated packages are downloaded and built again. Other packages should be reused if it is already installed once. 

This can be achieved with docker images, where docker build uses images from previous builds if they exist.

The effort going into minti tests will take this into consideration. Right now, this is one step towards that.

Reply:
Yep, this is the right approach